components/admin/Textinput.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styles from "@styles/admin.module.css";
+
+type Props = {
```

Comment:
Could you follow in the typing for every component what we use to do everywhere so here we would have 

`type TextInputProps`

Reply:
Now that I see this I see that most of the component you made do `type Props` could you change that everywhere ? Thx !